packages/entando-apimanager/modules/api/apiManager.js

```diff
@@ -142,6 +118,94 @@ const normalizeErrorMessage = (message) => {
   return 'serverError';
 };
 
+export const authIntercept = () => {
```

Comment:
why is this being exported?

Reply:
this is used in `current-user/actions.js` in order to save the new set of tokens into fetch-token-intercept after saving it in local storage. I noticed this because fetch-token-intercept does not provide option to save the new refresh token so I "re-establish" the token intercept props then. But I guess the real issue here is I think I will rename `authIntercept` to avoid confusion, making it related to what it actually does.